zone/aa.cpp

```diff
@@ -37,6 +37,12 @@ Copyright (C) 2001-2016 EQEMu Development Team (http://eqemulator.net)
 
 extern QueryServ* QServ;
 
+namespace detail
```

Comment:
Wouldn't we want to namespace this `AA` or something similar

Reply:
I prefer detail for non-public stuff personally.

Reply:
It would normally since the client i was using (RoF2) only goes up to 100.  I carefully crafted my base hp to not make it an issue though.